src/components/UserSearch.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import {Route} from 'react-router-dom';
+
+class UserSearch extends Component {
```

Comment:
Want to take a stab at testing this component?

Reply:
Yes! I can do some reading on jest so I have some idea when we pair.